astropy/modeling/functional_models.py

```diff
@@ -818,8 +818,8 @@ def fit_deriv(cls, x, x_0, amplitude_L, fwhm_L, fwhm_G):
         dVdx = np.sum((D/beta - 2 * (X - B) * alpha / np.square(beta)), axis=-1)
         dVdy = np.sum((C/beta - 2 * (Y - A) * alpha / np.square(beta)), axis=-1)
 
-        dyda = [constant * V / amplitude_L,
-                - constant * dVdx * 2 * sqrt_ln2 / fwhm_G,
+        dyda = [- constant * dVdx * 2 * sqrt_ln2 / fwhm_G,
```

Comment:
I guess one last nag would be that the exception is in this `-` sign right here, since it's a unary operator not a binary operator.  Kinda looks funny floating off by itself.  But I'll fix that later it's really no big deal and it seems to at least pass the tests.